hydra-cluster/hydra-cluster.cabal

```diff
@@ -146,10 +155,27 @@ library
 
   ghc-options:     -haddock
 
+executable hydra-cluster
+  import:             project-config
+  hs-source-dirs:     exe/hydra-cluster
+  main-is:            Main.hs
+  ghc-options:        -threaded -rtsopts
+  build-depends:
+    , base                  >=4.7 && <5
+    , hydra-cardano-api
+    , hydra-cluster
+    , hydra-node
+    , hydra-prelude
+    , hydra-test-utils
+    , optparse-applicative
+
+  build-tool-depends: hydra-node:hydra-node -any
+  ghc-options:        -threaded -rtsopts
+
 executable log-filter
   import:         project-config
-  hs-source-dirs: exe
-  main-is:        log-filter.hs
+  hs-source-dirs: exe/log-filter
+  main-is:        Main.hs
```

Comment:
Question: why not using the exe name as file name? I know we had this discussion already but cannot remember the rationale behind it.